openshift_metrics/invoice.py

```diff
@@ -78,7 +78,7 @@ def get_service_unit(self) -> ServiceUnit:
             SU_CPU: {"gpu": -1, "cpu": 1, "ram": 4},
             SU_A100_GPU: {"gpu": 1, "cpu": 24, "ram": 74},
             SU_A100_SXM4_GPU: {"gpu": 1, "cpu": 32, "ram": 245},
```

Comment:
On the NERC website, the RAM advertised for A100 SXM is 240GB. This shows 245GB. Was this intended?

Reply:
Good catch. While this won't matter I will update it to match the website.

Reply:
@QuanMPhm updated

Reply:
@QuanMPhm this is now aligned with the website, and I have updated the tests as well.